src/RolloutScorer/RolloutScorer/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+        public const string DowntimeLabel = "Rollout Downtime";
+
+        // The git file mode of a "file blob"; more info here: https://developer.github.com/v3/git/trees/#parameters
+        public const string GitFileMode = "100644";
+
+        public const string KeyVaultUri = "https://engkeyvault.vault.azure.net";
+        public const string GitHubPatSecretName = "BotAccount-dotnet-bot-repo-PAT";
+        public const string StorageAccountKeySecretName = "rolloutscorecards-storage-key";
+        public const string StorageAccountName = "rolloutscorecards";
+        public const string ScorecardsTableName = "scorecards";
+
+        public static bool IssueContainsRelevantLabels(Issue issue, string issueLabel, string repoLabel, ILogger log = null)
+        {
+            if (issue == null)
+            {
+                WriteWarning("A null issue was passed.", log);
+                return false;
+            }
+
+            return issue.Labels.Any(l => l.Name == issueLabel) && issue.Labels.Any(l => l.Name == repoLabel);
+        }
+
+        /// <summary>
+        /// Parse and return config
+        /// </summary>
+        /// <returns>Config object representing config</returns>
+        public static Config ParseConfig()
+        {
+            Config config;
+            string configFile = $"{AppContext.BaseDirectory}/config.json";
+            if (!File.Exists(configFile))
+            {
+                WriteError($"ERROR: Config file not found; expected it to be at '{configFile}'");
+                return null;
+            }
+            using (JsonTextReader reader = new JsonTextReader(new StreamReader(configFile)))
+            {
+                config = JsonSerializer.Create().Deserialize<Config>(reader);
+            }
+            return config;
+        }
+
+        public static CloudTable GetScorecardsCloudTable(SecretBundle storageAccountKey)
+        {
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(
+                connectionString: $"DefaultEndpointsProtocol=https;AccountName={StorageAccountName};AccountKey={storageAccountKey.Value};EndpointSuffix=core.windows.net");
+            CloudTableClient tableClient = storageAccount.CreateCloudTableClient();
+            return tableClient.GetTableReference(ScorecardsTableName);
+        }
+
+        public static GitHubClient GetGithubClient(SecretBundle githubPat)
+        {
+            ProductInfoHeaderValue productHeader = GetProductInfoHeaderValue();
+            GitHubClient githubClient = new GitHubClient(new Octokit.ProductHeaderValue(productHeader.Product.Name, productHeader.Product.Version))
+            {
+                Credentials = new Credentials("fake", githubPat.Value)
+            };
+
+            return githubClient;
+        }
+
+        public static ProductInfoHeaderValue GetProductInfoHeaderValue()
+        {
+            return new ProductInfoHeaderValue(typeof(Program).Assembly.GetName().Name,
```

Comment:
Since this is tied to typeof(Program), I'd probably make it a static method on Program.  Otherwise if someone moves this method (not noticing the implicit Program dependency here), we might get some weird behavior.

Reply:
Smart and done.